jexl-eval/src/lib.rs

```diff
@@ -651,4 +687,131 @@ mod tests {
             value!([{"bobo": 50, "fofo": 100}, {"bobo": 60, "baz": 90}])
         );
     }
+
+    #[test]
+    fn test_binary_equals_and_not_equals() {
```

Comment:
I'm curious about your choice of the word `binary` for this and the next test that you added. Is it a reference to the argument count for what is being tested (i.e. testing _two_ arguments for equality/inequality)?  Just a very uber nit, but the word choice made me assume there were going to be some bitwise magic happening

Reply:
These are operators with two operands.

Note to self: we need to implement unary operators `-`, `+` and `!`.

Reply:
Ahh, that makes sense now!